app/timetables/models.py

```diff
@@ -39,3 +39,21 @@ def clean(self):
 
     def __str__(self):
         return self.name
+
+
+class Dish(ForceCapitalizeMixin, TimestampMixin):
```

Comment:
@andela-joyebanji: Though `TimestampMixin` extended `models.Model`, I'm still of the opinion, as @andela-uawili stated, that all models should have `models.Model` as their base class regardless of inheriting from a class that have extended it.